lib/metadata/index.js

```diff
@@ -41,7 +41,9 @@ module.exports = function getMetadada(number, author, content) {
         }).then(function(owners) {
             return status(owners);
         }).then(function(owners) {
-            metadata.owners = owners;
+            metadata.owners = owners.filter(function(owner) {
```

Comment:
There are *lots* of places where I could put this filtering. I picked between as early as possible (here) or as late as possible, in `getReviewers`, and picked this one. Happy to pick any other place.